src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -32,6 +33,7 @@ public function getFunctions()
     {
         return array(
             new \Twig_SimpleFunction('profiler_dump', array($this, 'dumpValue')),
+            new \Twig_SimpleFunction('http_status_text', array($this, 'httpStatusText')),
```

Comment:
your function depends only on HttpFoundation, so it should go in the HttpFoundationExtension of the bridge IMO